create_seed_checkpoint.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
```

Comment:
Can we add some comments in the beginning of the file to explain what this script is used for and how to run this?